mmd_tools/operators/fileio.py

```diff
@@ -260,6 +260,9 @@ def execute(self, context):
             bpy.ops.pose.transforms_clear()
             bpy.ops.object.mode_set(mode='OBJECT')
             root.mmd_root.show_armature = prev_show
+        # Add extension if needed
+        if not self.filepath.endswith(self.filename_ext):
+            self.filepath += self.filename_ext 
```

Comment:
Actually it seems not... Oh! I have noticed that the latest version on master was [incorrectly using](https://github.com/powroupi/blender_mmd_tools/blob/f5591f331aa1359bfe755451f2f28c45fdfe4732/mmd_tools/operators/fileio.py#L168) the `ImportHelper`. I was using the old version when I found this bug.